### DISCUSSION TITLE
Method of detecting that quick select mode is active?

### DISCUSSION THREAD

**Discussion by neogeographica** [wezterm/wezterm#6098] [Q&A]
### What Operating System(s) are you running on?

Linux X11, macOS

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

I was noodling with some status bar changes to highlight when a mode is active that changes the keybinds, e.g. copy or search or quick select.

For copy or search mode I can detect when they are active by looking at the active key table. For quick select mode though, no such luck. Is there any state I can observe from Lua to see if that mode is active?

**Comment by curbol** [comment]
I'm having the same issue. Not able to detect if quick-select mode is active.
